timesketch/lib/utils.py

```diff
@@ -189,9 +198,9 @@ def read_and_validate_redline(file_handle):
         row_to_yield["timestamp"] = timestamp
         row_to_yield["datetime"] = dt_iso_format
         row_to_yield["timestamp_desc"] = timestamp_desc
-        row_to_yield["alert"] = alert #extra field
+        row_to_yield["alert"] = alert  # Extra field
```

Comment:
I know thi sis not from this this PR, but can you change all these double quotes to a single one (" vs ') to be consistent with the rest of the codebase